app/views/applications/show.html.erb

```diff
@@ -20,3 +20,13 @@
     <% end %>
   <% end %>
 <% end %>
+
+<% if @application.pets.count > 0 && @application.status == "In Progress"%>
+  <%= "Submit Application" %>
+  <%= form_with url: "/applications/#{@application.id}?status=Pending", method: :patch, params: {application_id: @application.id}, local: true do |form| %>
```

Comment:
Nice knowing that we can use query a certain param we want to send to the Controller.